Extensions/addressbar.js

```diff
@@ -24,16 +24,15 @@ XKit.extensions.addressbar = new Object({
 	run: function() {
 		this.running = true; //there's some confusion over whether this gets set to true on immediate return
 		
-		let XInterface = XKit.interface.where()
-		if (XInterface.dashboard !== true || XInterface.endless !== true) {
+		const Xwhere = XKit.interface.where();
+		if (Xwhere.dashboard !== true || Xwhere.endless !== true) {
 			return; //only useful on endless dashboard
 		} 
 
-		XKit.extensions.addressbar.interval = setInterval(
-			XKit.extensions.addressbar.update_address, 500);
+		this.interval = setInterval(this.update_address, 500);
 
-		XKit.extensions.addressbar.initialScroll = history.scrollRestoration;
-		history.scrollRestoration = "manual";
+		this.initialScroll = history.scrollRestoration;
+		history.scrollRestoration = "manual"; //prevent page from jumping to bottom on refresh 
```

Comment:
\*

Reply:
This one's an experimental API that isn't quite self-evident why it's used, so I'd rather leave it